### ISSUE TITLE
Issue in forceFitColumns option

### ISSUE THREAD

**Issue by nzhtrtlc** [6pac/SlickGrid#248]
Hi, there is an issue in forceFitColumns option or this.grid.autoSizeColumns();
In my grid which have lot of columns i want to fit my columns but there is an style issue about it.
Here is the screen record for you https://drive.google.com/file/d/1zuYPKvU02OFepvjeS5fvoJh3G0BG2Iep/view

I have no problem with less columns, but with lot of columns it brokes like in video.

**Comment by nzhtrtlc** [comment]
@6pac You are right the problem is in my own code. My grid is in my own tab component 
(i am usign react) When i take my grid out of the tab component it works great but when i put it back to tab component it brokes as in the video record. I have no idea why is happening :(

**Comment by ghiscoding** [comment]
Could that be a problem because of #239 ?

**Comment by 6pac** [comment]
It's not throwing any errors in the console? If it was #239, could the issue still occur if there was no dragging and dropping attempted in the repro? I'm assuming overriding the events could cause errors, but they'd presumably be visible as an indicator.

**Comment by 6pac** [comment]
I think reworking SlickGrid to remove the jQuery drag and drop dependencies is a great idea. HOWEVER, articles like [this one](https://www.quirksmode.org/blog/archives/2009/09/the_html5_drag.html) scare the hell out of me. Seems no-one has posted much since 2010 - there's a lot of talk from that year (eg. [here](https://www.html5rocks.com/en/tutorials/dnd/basics/), [here](https://www.useragentman.com/blog/2010/01/10/cross-browser-html5-drag-and-drop/) and more recent but less complete [here](https://github.com/leonadler/drag-and-drop-across-browsers) ), but I can't find anywhere where someone has said 'this is all resolved now'. So I'm forced to conclude that we're opening a big time can of worms trying to do that - and that's what the contributor who originally tried and failed to do it said: see https://github.com/6pac/SlickGrid/issues/14 https://github.com/6pac/SlickGrid/issues/15

**Comment by 6pac** [comment]
OK @nzhtrtlc, I think that's the best for now. @ghiscoding is right, this problem may be solved on a different branch, I've just got to check it out and work out how that fits in with the main repo.
